getting-started.md

```diff
@@ -12,41 +12,64 @@
 
 **Step 2** - Deploy the vCenter Event Broker Appliance OVA to your vCenter Server using the vSphere HTML5 Client. As part of the deployment you will be prompted to provide the following input:
 
-*Networking*
+*Networking* (**Required**)
 
   * Hostname - The FQDN of the vCenter Event Broker Appliance. If you do not have DNS in your environment, make sure the hostname provide is resolvable from your desktop which may require you to manually add a hosts entry. Proper DNS resolution is recommended
   * IP Address - The IP Address of the vCenter Event Broker Appliance
-  * Netmask Prefix - CIDR Notation (e.g. 24 = 255.255.255.0)
+  * Network Prefix - Network CIDR Selection (e.g. 24 = 255.255.255.0)
   * Gateway - The Network Gateway address
   * DNS - DNS Server(s) that will be able to resolve to external sites such as Github for initial configuration. If you have multiple DNS Servers, input needs to be space separated.
   * DNS Domain - The DNS domain of your network
+  * NTP Server - NTP Server(s) for proper time synchronization. If you have multiple DNS Servers, input needs to be space separated.
 
-*Credentials*
+*Proxy Settings (Optional)*
+  * HTTP Proxy Server - HTTP Proxy Server followed by the port and without typing http:// before (e.g. proxy.provider.com:3128)
+  * HTTPS Proxy - HTTPS Proxy Server followed by the port and without typing https:// before (e.g. proxy.provider.com:3128)
+  * Proxy Username - Optional Username for Proxy Server
+  * Proxy Password - Optional Password for Proxy Server
+  * No Proxy - Exclude internal domain suffix. Comma separated (localhost, 127.0.0.1, domain.local)
 
+*OS Credentials* (**Required**)
   * Root Password - This is the OS root password for the vCenter Event Broker Appliance
-  * OpenFaaS Password - This is the Admin password for OpenFaaS UI
 
-*vSphere*
+
+*vSphere* (**Required**)
 
   * vCenter Server - This FQDN or IP Address of your vCenter Server that you wish to associate this vCenter Event Broker Appliance to for Event subscription
   * vCenter Username - The username to login to vCenter Server, as mentioned earlier, readOnly account is sufficient
   * vCenter Password - The password to the vCenter Username
   * Disable vCenter Server TLS Verification - If you have a self-signed SSL Certificate, you will need to check this box
 
-*zDebug*
+*Event Processor Configuration* (**Required**)
+  * Event Processor - Choose either OpenFaaS (default) or AWS EventBridge and only fill in the configuration for the selected event processor
+
+*OpenFaaS Configuration* (**Required if selected as Event Processor**)
+  * Password - Password to login into OpenFaaS. Please use a secure password
+  * Advanced Settings - N/A, future use
```

Comment:
"async" fn invocation mode is already a supported advanced setting today. If you want to provide an example:

```json
"options": {
            "async": "true"
        }
```

Reply:
Was going to defer this until we had more Adv options, since we didn't have any for AWS EventBridge, was just setting up the OVF template to support this in the future